Index: src/main/resources/assets/standardblt/lang/zh_cn.lang
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>item.bltItem.name=标准的BLT物品
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/assets/standardblt/lang/zh_cn.lang b/src/main/resources/assets/standardblt/lang/zh_cn.lang
--- a/src/main/resources/assets/standardblt/lang/zh_cn.lang	
+++ b/src/main/resources/assets/standardblt/lang/zh_cn.lang	
@@ -1,1 +1,13 @@
-item.bltItem.name=标准的BLT物品
\ No newline at end of file
+############
+#StandardBLT
+############
+
+# Items
+item.BLTItem.name=标准的BLT物品
+## Food
+item.BLTFood.name=标准的BLT食物
+item.BLTEffectFood.name=标准的BLT效果
+# Blocks
+tile.BLTBlock.name=标准的BLT方块
+# Creative Tab(s)
+itemGroup.StandardBLT_Tab=标准的BLT
\ No newline at end of file
